src/REPLMode.jl

```diff
@@ -167,6 +169,8 @@ const help = Base.Markdown.parse("""
     `rm`: remove packages from project or manifest
 
     `up`: update packages in manifest
+
+    `init` initializes a project in the current, or git base, directory
```

Comment:
Different scenario: what if you have `JULIA_ENV` set to a name like `v1.2.3` and `~/.julia/environments/v1.2.3` does not yet exist. Should this create those files? In that case it's more reasonable to just auto-create the directory and its project and manifest files. But what if someone explicitly does `pkg> init` in those circumstances? Or maybe all environments should be explicitly created...

Reply:
If you have an explicit `JULIA_ENV` (that is not `.`) then that will be automatically initialized on any pkg command. This PR should only change the behavior of creating new environments in the current working directory, opting to require an explicit initialization in those cases:

```
shell> ls .git
HEAD        config      description hooks       info        objects     refs

julia> import Pkg3

julia> ENV["JULIA_ENV"] = "v1.2.3"
"v1.2.3"

pkg> st
INFO: Status "~/.julia/environments/v1.2.3/Project.toml"

julia> delete!(ENV, "JULIA_ENV");

pkg> st
INFO: Status "~/.julia/environments/v0.6/Project.toml"

julia> ENV["JULIA_ENV"] = "."
"."

pkg> st
Could not find local environment in /Users/kristoffer/Documents/gitrepo, do you want to create it?
 > yes
   no
INFO: Initialized project in /Users/kristoffer/Documents/gitrepo by creating the file Project.toml
INFO: Status "Project.toml"
```

Reply:
I assume since you're making the change that this approach "feels right"?